### PR TITLE
masscan: 1.3.0 -> 1.3.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 1.3.1

Change log: https://github.com/robertdavidgraham/masscan/releases/tag/1.3.1

Also, ran `nixpkgs-fmt`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 110894` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>masscan</li>
  </ul>
</details>

File: pkgs/tools/security/masscan/default.nix [hunk 1]
```diff
@@ -34,9 +34,10 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Fast scan of the Internet";
-    homepage    = "https://github.com/robertdavidgraham/masscan";
-    license     = licenses.agpl3;
-    platforms   = platforms.unix;
+    homepage = "https://github.com/robertdavidgraham/masscan";
+    changelog = "https://github.com/robertdavidgraham/masscan/releases/tag/${version}";
+    license = licenses.agpl3;
```
Comment (SuperSandro2000) [inline pkgs/tools/security/masscan/default.nix:39]: agpl3Plus or agpl3Only?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/security/masscan/default.nix (hunk 1 above)
Comment (rnhmjoj) [reply pkgs/tools/security/masscan/default.nix:39]: Seems 3 only: https://raw.githubusercontent.com/robertdavidgraham/masscan/master/LICENSE

Comment (fabaff) [reply pkgs/tools/security/masscan/default.nix]: Thanks for the feedback, guys.

Updated.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 110894` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>masscan</li>
  </ul>
</details>

**Review by rnhmjoj — APPROVED** [review]
Looks like this release is mostly bugfixes. I tested masscan with
```
# result/bin/masscan -p 80 0.0.0.0/0 --excludefile result/etc/masscan/exclude.conf
```
and seems to work all right.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110894` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>masscan</li>
</ul>
</details>

**Pull request merged**